src/mapnik_image.cpp

```diff
@@ -2546,7 +2546,7 @@ NAN_METHOD(Image::fromSVGBytesSync)
  * @param {Object} [options]
  * @param {number} [options.scale] - scale the image. For example passing `0.5` as scale would render
  * your SVG at 50% the original size.
- * @param {number} [options.max_size] - the maximum allowed size of the svg dimensions * scale. The default is 2048.
+ * @param {number} [options.max_size] - the maximum allowed size of the svg area at the provided scale. The default is 4194304.
```

Comment:
I've changed the default min from 2048 to 4194304 (2048^2) as 2048 was the max size of an edge before this change. This should mean that images that use to pass by default, will still pass.